pkgs/development/libraries/librealsense/default.nix

```diff
@@ -80,8 +80,13 @@ stdenv.mkDerivation rec {
   postInstall = ''
     substituteInPlace $out/lib/cmake/realsense2/realsense2Targets.cmake \
     --replace-fail "\''${_IMPORT_PREFIX}/include" "$dev/include"
-  '' + lib.optionalString enablePython  ''
+  '' + lib.optionalString enablePython ''
     cp ../wrappers/python/pyrealsense2/__init__.py $out/${pythonPackages.python.sitePackages}/pyrealsense2
+  '' + ''
+    # install udev rules
+    # based on scripts/setup_udev_rules.sh
+    install -Dm644 -t $out/lib/udev/rules.d/ ../config/99-realsense-libusb.rules
```

Comment:
Looking at the installed files, there are references to FHS path like `/bin/bash` and `/usr/local/bin/usb-R200-in_udev` and also expects `rs_ipu6_d457_bind.sh` on `PATH` so this will not be enough.

Reply:
you're right, i didn't consider that the rules would want to shell out